### ISSUE TITLE
2021-3-12

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#478]
741Figure out the difference between import and from import; (2021-3-5)
742Try to delete torch.nn.functional as F; done; (2021-3-5)
743Try to delete dataset in trainer.py; done; (2021-3-5)
744We can open two projects in two windows for pycharm; (2021-3-5)
745Compute position of each element; done; (2021-3-5)
746Delete dataset in tester; done; (2021-3-5)
747Try to read tutorials of pytorch; 所有的模型都需要继承torch.nn.Module ， 需要实现以下⽅法; super必然在此中; forward函数也在此中; done; (2021-3-5)
748Put loss function in front of optimizer; done; (2021-3-5)
749(1) load data (in class Dataset); (2) torch.nn.Module, forward function (in class Model); (3) loss function, optimizer (in file trainer); (4) iteration (in file trainer); (2021-3-5)
